test/Microsoft.NET.TestFramework/CoreMSBuildAndWindowsOnlyTheoryAttribute.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.NET.TestFramework
+{
+    public class CoreMSBuildAndWindowsOnlyTheoryAttribute : TheoryAttribute
```

Comment:
Can we do that in a separate PR though? I would like to merge this one and then we can bring that over.

I agree that's much better.